src/l500/l500-color.cpp

```diff
@@ -485,13 +485,15 @@ namespace librealsense
                 }
 
                 try {
-                    // endpoint 5 - 32KB
-                    command cmdTprocGranEp5(ivcam2::TPROC_USB_GRAN_SET, 5, usb_trb);
-                    _owner->_hw_monitor->send(cmdTprocGranEp5);
-
-                    command cmdTprocThresholdEp5(ivcam2::TPROC_TRB_THRSLD_SET, 5, 1);
-                    _owner->_hw_monitor->send(cmdTprocThresholdEp5);
-
+                    // Keep the USB power on while triggering multiple calls on it.
+                    _owner->get_raw_depth_sensor().invoke_powered([&](platform::uvc_device& dev) {
```

Comment:
I don't like manual access to `_owner->get_raw_depth_sensor()` -- it might change, and we might want to do other stuff in the future.

I think we should add a utility function that inside will know how to raise power... `group_hw_monitor_calls` or something along those lines.

Reply:
Done